internal/cache/metadata/stat_cache_test.go

```diff
@@ -325,54 +325,54 @@ func (t *MultiBucketStatCacheTest) CreateEntriesWithSameNameInDifferentBuckets()
 	ExpectTrue(spices.NegativeEntry("apple", justBefore))
 }
 
-func (t *MultiBucketStatCacheTest) FillUpToCapacity() {
```

Comment:
This test has been disabled as it's not straight-forward anymore to predict the actual-size and thus evictions etc. This test needs actual size to be calculated and used in the test.